### DISCUSSION TITLE
Youtubeや外部ブログへの誘導のための宣伝投稿について

### DISCUSSION THREAD

**Discussion by mangofruits2024** [increments/qiita-discussions#604] [要望・アイデア・不具合]
投稿の内容についてです。
昨今、自身のYoutubeへの登録数、再生数を増やすための宣伝
ブログなどへの外部リンク
ご自身の所属会社の宣伝投稿
のような内容が増えてきているように思います。

まだ内容を抜粋しているものはマシな方で
明らかにブログの目次代わりにしてQiitaへ投稿している、
あからさまな自社・Youtubeなどの宣伝という内容のものが増えているように感じます。

Qiita Advent Calendarでは、
＞QiitaはもちろんZennやnote、はてなブログ、個人ブログなど、エンジニアに関わる内容であればどのサイトの記事でも歓迎です。

とあるため可能かもしれませんが、基本的な投稿場面でもこのような状況は容認されているのでしょうか。
ガイドラインでは、エンジニアに関する知識を記録・共有するためのサービスと定義されていると思います。
そのため、こういった宣伝利用が増えていくことに疑問に感じました。

ちなみにリンクしかないような記事は通報はしておりますが、多くないためか消えないことの方が多いです。

**Comment by tanaka-asami415** [comment]
@mangofruits2024 
ご報告ありがとうございます🙏

記事が外部リンクのみになっていたり、記事の内容と直接的に関係ないURLに誘導したり、
記事上で完結させることも出来る情報を敢えて載せずに、外部のサイトに誘導することは利用規約違反となりますので順次対応を行っております。

上記に該当しないような内容については、コミュニティガイドラインに沿った対応を行っております。
[コミュニティガイドライン違反の記事に対する運営の方針について](https://help.qiita.com/ja/articles/qiita-community-guideline#part-cf79556a40a47ce0)

原則としてユーザーさまからの通報を元に当社で検討・判断をし、修正の依頼をする等の対応を行なっております。
1ユーザーさまからの通報だけでなく、複数アカウントからの通報等も合わせての総合判断となります🙇

運営の強権によって管理するのではなく、コミュニティの反応をお伝えして、気づきを与えることが運営の役割だと考えているため、
強制的な対応をしない場合でも、コミュニティガイドラインに沿って、運営からユーザー様への警告等の対応を順次行っています。

対応しきれていない部分もあり、ご不便をおかけして申し訳ありませんが、引き続き[通報機能](https://help.qiita.com/ja/articles/qiita-report)よりご報告いただけますと幸いです🙇
